src/data_vis.ts

```diff
@@ -354,6 +354,16 @@ class DataVisModel {
         });
     }
 
+    private exportAllData() {
+        const fileName : string = reportInfo.dep_id + "_data_set.zip"
+        let a = document.createElement("a");
+        document.body.appendChild(a);
+        a.href = "data_set.zip";
+        a.download = fileName;
+        a.click();
```

Comment:
What happens if the file doesn't exist? i.e. if the montagu report hasn't been set up correctly and it fails to create the file. Do we need to put some nicer handling in for that case? Even if it is just not crashing the UI and logging the error somewhere?

Reply:
If the report does not create a `data_set.zip` file then orderly will throw an error.

So the case we are protecting ourselves against is the report had been specifically modified so that  `data_set.zip` is not created or expected by orderly. [or orderly not working correctly]

It might worth having a list of files that have to exist the gets checked as part of `npm run build`.

Reply:
Okay great, I will approve this then. If you think it is worth adding those checks to the `npm run build` let's do that in a separate ticket.